<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="Jatinder" failures="1" tests="6" name="Regression(failed)" time="213.799" errors="0" timestamp="2024-03-12T11:38:58 MDT">
  <testcase classname="amazonST.Testaddress" name="Invaliddata" time="21.871"/>
  <testcase classname="amazonST.Testaddress" name="security" time="34.083"/>
  <testcase classname="amazonST.TestSign" name="validdata" time="10.677"/>
  <testcase classname="amazonST.TestSign" name="invalid" time="5.908"/>
  <testcase classname="amazonST.Testsearch" name="validdata" time="23.844"/>
  <testcase classname="amazonST.Testsearch" name="count" time="9.216">
    <failure type="java.lang.AssertionError" message="expected [60 products] but found [1-48 of over 50,000 results for]">
      <![CDATA[java.lang.AssertionError: expected [60 products] but found [1-48 of over 50,000 results for]
at amazonST.Testsearch.count(Testsearch.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- count -->
  <testcase classname="amazonST.TestSign" name="check" time="0.0">
    <ignored/>
  </testcase> <!-- check -->
  <testcase classname="amazonST.Testaddress" name="validdata" time="0.0">
    <ignored/>
  </testcase> <!-- validdata -->
</testsuite> <!-- Regression(failed) -->
